docs/guides/docker.md

```diff
@@ -0,0 +1,256 @@
+---
+label: Docker automation
+order: -2
+description: Learn how to deploy a Docker stack for common media automation software.
+image: /static/tohsaka.gif
+---
+
+# Setting up a Docker automation stack
+
+Docker is a piece of software that allows you to virtualize applications into containers, so that you don't have to deal with dependency
+management and can carry around a solution that's easily deployable anywhere you go.
+
+This guide will cover how to setup a Docker stack for common media automation programs, namely: a torrent client, a usenet client, a
+VPN with a killswitch, Sonarr, Radarr, and Jackett.
+
+## Installing Docker
+
+First we need to install Docker onto our system. Windows users can do so by downloading
+[Docker Desktop](https://docs.docker.com/desktop/install/windows-install/), while Linux users can follow the instructions outlined
+[here](https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository), after which they can proceed onto the
+[post-installation steps](https://docs.docker.com/engine/install/linux-postinstall).
+
+## Setting up a Docker network
+
+Now we'll be setting up a network so that our containers can communicate with each other. This step's quite simple, it's just `docker network create guide`. A list of all your current networks can be seen with `docker network ls`.
+
+## Generating a Wireguard configuration file
+
+For maximum privacy and security we'll be tunnelling our usenet and torrent clients through a VPN. Radarr, Sonarr and Jackett however won't go
```

Comment:
Probably worth mentioning Prowlarr?

Reply:
added a mention of prowlarr as a jackett alternative.